PWGDQ/Core/VarManager.h

```diff
@@ -1027,6 +1051,106 @@ void VarManager::FillPairVertexing(C const& collision, T const& t1, T const& t2,
   }
 }
 
+template <uint32_t collFillMap, uint32_t fillMap, typename C, typename T1, typename T2, typename T3>
+void VarManager::FillDileptonLepton(C const& collision, T1 const& lepton1, T2 const& lepton2, T3 const& lepton3, float* values)
```

Comment:
I think we have to generalize a bit this function to be used by both muon, barrel or muon-barrel analyses, similar to the FillPairVertexing() function.
I am not sure why you need to have all the T1, T2 and T3 typenames. At the moment you use just a single type (all 3 particles are muons). I think that to make this general, you could us just two types, T1 and T2, where T1 is for the tracks that make up your dilepton (either track or fwdtrack), and T2 can be the third track, which can be independently from T1, either a fwd of or a barrel track.
I suggest to also change the name of the function, to something more general since the same functionality is needed also for J/psi + K. So i suggest to change to FillDileptonTrackVertexing maybe?